### DISCUSSION TITLE
Unbreakable egg

### DISCUSSION THREAD

**Discussion by hilltty** [PurpurMC/Purpur#1259] [__Completed__] [answered]
Add a setting in the config to make the dragon egg unbreakable, i.e. set its durability to -1, this will help people who want to keep the egg in its place

**Comment by RhythmicSys** [comment, marked as answer]
There is an option for [Dragon Egg teleportation](https://purpurmc.org/docs/Configuration/dragon_egg), but if you want the dragon egg to be completely unbreakable, you can add it to the "unbreakable blocks" tag in a datapack
